lib/nero/wildlife_watch/wildlife_watch_classification.rb

```diff
@@ -22,7 +22,7 @@ def annotations
 
       def task
         case workflow_id
-        when "1021"
+        when "1021", "1717"
```

Comment:
I also want to move the config to be in the workflow retirement column rather than inside Nero.